sds_data_manager/constructs/compute_resources.py

```diff
@@ -204,3 +204,101 @@ def __init__(
             },
             tags={"Purpose": "Batch Processing"},
         )
+
+
+class IalirtEC2Resources(Construct):
```

Comment:
I'd suggest moving everything into its own file like `ialirt_stack.py` so this can really be separated more easily.